kala.py

```diff
@@ -213,8 +233,11 @@ def get(mongodb, collection):
     return {'results': [document for document in cursor]}
 
 
-@app.route('/<collection>', method=['POST'])
+@app.route('/<collection>', method=['POST', 'OPTIONS'])
 def post(mongodb, collection):
+    if not app.config['cors.enable']:
```

Comment:
Doesn't this want to be `if request.method == 'POST' and not app.config['cors.enable']:` otherwise it'll abort on everything, not just `OPTIONS`.